doc/man1/flux-uri.rst

```diff
@@ -143,6 +157,15 @@ Get the URI of a local flux-broker
    $ flux uri pid:$(pidof -s flux-broker)
    local:///tmp/flux-sLuBkZ/local-0
 
+The following submits a batch job and returns the instance URI once it is
+available. Without the :option:`--wait` flag, :command:`flux uri` would fail
+immediately with "job is not running":
+
+::
+
+   $ flux uri --wait $(flux batch -n1 --wrap flux run hostname)
```

Comment:
In this example, maybe it would be slightly better to use `sleep 3600` or something that runs for a while rather than `hostname`?

Reply:
Good point! If we're making a realistic example, maybe just a placeholder `myjob` or something?

Reply:
Sounds fine!